src/app/externs/backendapi.js

```diff
@@ -78,3 +78,18 @@ backendApi.ReplicaSet;
  * }}
  */
 backendApi.NamespaceList;
+
+/**
+ * @typedef {{
+ *   labels: !Array<!backendApi.Label>
+ * }}
+ */
+backendApi.LabelsList;
```

Comment:
I think you don't need this object; you can safely remove it from this change. Then, when implementing backend, you can add `labels: !Array<!backendApi.Label>` property to `AppDeployment`